main.lua

```diff
@@ -22,6 +24,8 @@ local d = Babble.dialogue()
       :print("Strawberry is really nice, yeah.")
    :endNode()
 
+   :switch('start')
```

Comment:
I could have `:pushed('start')` instead which would have been better.

But switch lets you "reset" any dialogue to a specific Node which is good